snuba/subscriptions/worker.py

```diff
@@ -78,22 +129,24 @@ def flush_batch(self, batch: Sequence[Sequence[SubscriptionResultFuture]]) -> No
             future.result()
 
 
-class SubscriptionResultCodec(Codec[KafkaPayload, SubscriptionResult]):
-    def encode(self, value: SubscriptionResult) -> KafkaPayload:
+class SubscriptionTaskResultCodec(Codec[KafkaPayload, SubscriptionTaskResult]):
+    def encode(self, value: SubscriptionTaskResult) -> KafkaPayload:
         subscription_id = str(value.task.task.identifier)
+        request, result = value.result
         return KafkaPayload(
             subscription_id.encode("utf-8"),
             json.dumps(
                 {
-                    "version": 1,
+                    "version": 2,
```

Comment:
Is the consuming code aware of this change already ?

Reply:
I've talked with @wedamija about it but I will need to wait until he's ready to merge this.